scripts/lint/lint-markdown.js

```diff
@@ -13,6 +13,14 @@ const path = require("path");
 const FRONT_MATTER_REGEX = /((^---\s*$[^]*?^---\s*$)|(^\+\+\+\s*$[^]*?^(\+\+\+|\.\.\.)\s*$))(\r\n|\r|\n|$)/m;
 const AUTO_GENERATED_HEADING_REGEX = /###### Auto generated by ([a-z0-9]\w+)[/]([a-z0-9]\w+) on ([0-9]+)-([a-zA-z]\w+)-([0-9]\w+)/g;
 
+/**
+ * REGEX for matching top-level provider index files only. Example:
+ *
+ * registry/themes/default/content/registry/packages/auth0/_index.md
+ *
+ */
+const PACKAGE_INDEX_REGEX = /packages\/[a-z0-9]*\/_index.md/
```

Comment:
```suggestion
const PACKAGE_INDEX_REGEX = /packages\/[a-z0-9\-]*\/_index.md/
```

slight bug with the regex, since it is failing to match package names that have a hyphen in them, like `aws-native` and `azure-native`. Other than that though it is all working well!

Reply:
good call, tysm!